pkg/policy/checker.go

```diff
@@ -15,11 +15,6 @@ var (
 	EnvironmentsAndReleasesWithinSameYamlPartErr = errors.New("environments and releases cannot be defined within the same YAML part. Use --- to extract the environments into a dedicated part")
 	topConfigKeysRegex                           = regexp.MustCompile(`^[a-zA-Z]+:`)
 	separatorRegex                               = regexp.MustCompile(`^--- *$`)
-	topkeysPriority                              = map[string]int{
```

Comment:
maybe we can keep bases & environments with the same priority ?

Reply:
yeah. please have a try.

Reply:
Implemented as requested